### ISSUE TITLE
Style/Lambda doesn't work well for multi-line Rails named scopes

### ISSUE THREAD

**Issue by hopsoft** [testdouble/standard#21]
Consider the following named scope in a Rails model.

```ruby
class User < ApplicationRecord
  scope :with_name, ->(value) do
    first_name, last_name = value.to_s.split(" ")
    where first_name: first_name, last_name: last_name
  end
end
```

This produces the following offense.

```
Style/Lambda: Use the lambda method for multiline lambdas.
```

The required fix doesn't seem better than the original.

```ruby
class User < ApplicationRecord
  scope :with_name, (lambda do |value|
    first_name, last_name = value.to_s.split(" ")
    where first_name: first_name, last_name: last_name
  end)
end
```

**Comment by hopsoft** [comment]
It's not just the extra word. Ruby's parser doesn't understand the named scope without wrapping the lambda in parenthesis. Without the parens you'll get the following runtime error.

```
`lambda': tried to create Proc object without a block 
```

**Comment by searls** [comment]
Landed in 0.0.11
